src/commands/eval/evalAction.ts

```
@@ -0,0 +1,672 @@
+      errorsOnly: cmdObj.filterErrorsOnly,
+      firstN: cmdObj.filterFirstN,
+      metadata: cmdObj.filterMetadata,
+      pattern: cmdObj.filterPattern,
+      sample: cmdObj.filterSample,
+    };
+
+    testSuite.tests = await filterTests(testSuite, filterOptions);
+
+    if (
+      config.redteam &&
+      config.redteam.plugins &&
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)